### PR TITLE
Document that SDK 2.1 is now a pre-requisite for building

### PR DESCRIPTION
**Pull request by jcouv**
I'm following up with Lee to confirm that 2.1 bits will appear on the [official page](https://www.microsoft.com/net/download/core) and the [archive](https://github.com/dotnet/core/blob/master/release-notes/download-archive.md). Then we can remove the direct links.

### CODE REVIEW

**Comment by jcouv** [comment]
@jaredpar @jasonmalinowski @dotnet/roslyn-infrastructure for review. Updating contributing docs for windows.

File: docs/contributing/Building, Debugging, and Testing on Windows.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Using the command line Roslyn can be developed using the following pattern:
 1. [Visual Studio 2017 Update 3](https://www.visualstudio.com/vs/)
     - Ensure C#, VB, MSBuild and Visual Studio Extensibility are included in the selected work loads
     - Ensure Visual Studio is on Version "15.3" or greater
-1. [.NET Core SDK 2.0](https://www.microsoft.com/net/download/core)
+1. [.NET Core SDK 2.1](https://www.microsoft.com/net/download/core) (here are direct links while the document is getting updated: [Windows x64 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.1-preview-007094/dotnet-sdk-2.1.1-preview-007094-win-x64.exe), [Windows x86 installer](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.1-preview-007094/dotnet-sdk-2.1.1-preview-007094-win-x86.exe))
```
Comment (jasonmalinowski) [inline docs/contributing/Building, Debugging, and Testing on Windows.md]: Are these anticipated to be the final build? Should we rephrase the parenthetical to say "if you don't see them, the current previews are...)

**Comment by jcouv** [comment]
@jasonmalinowski Good point. Thanks
I'm having trouble retargeting, so I'll create a new PR.

**Comment by jasonmalinowski** [comment]
@jcouv You just needed to rebase your branch...

**Comment by jcouv** [comment]
I did, but then I would need to do a direct push to the dotnet repo, so that I could update the PR, but my enlistment is not set up for that.

**Pull request closed without merging**